pyshortcuts/shortcut.py

```diff
@@ -70,12 +70,20 @@ def __init__(self, script, name=None, description=None,
         if self.description is None:
             self.description = name
 
-        desktop = dest = os.path.join(get_homedir(), 'Desktop')
-        if folder is not None:
-            if folder.startswith(desktop):
-                folder = folder[len(desktop)+1:]
-            dest = os.path.join(desktop, folder)
-
+        # desktop = dest = os.path.join(get_homedir(), 'Desktop')
+        # if folder is not None:
+        #     # if folder var has a value
+        #     #   and it matches the beginning of just created `desktop` var
+        #     #   reassign folder var to be substring of folder var
+        #     #   that is 1 char longer that desktop var
+        #     # reassign `dest` to be desktop path + folder path
+        #     if folder.startswith(desktop):
+        #         folder = folder[len(desktop)+1:]
+        #     dest = os.path.join(desktop, folder)
+
+        #ignore all above, just make dest same as folder:
```

Comment:
I had a really hard time understanding what challenge this piece of code was solving, and in the end it seems to be not necessary at all. But maybe it's only unnecessary on Windows?